test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Use_Growth_Factors/dimensions.toml

```diff
@@ -83,12 +83,8 @@ Model years, 2013-2050, trivial, single record
 
 [[dimensions]]
 type = "time"
-class = "AnnualTime"
-time_type = "annual"
+class = "TrivialTime"
+time_type = "trivial"
```

Comment:
Maybe these should be "Constant" or "ConstantAnnual" instead of "trivial"?

Reply:
I'm actually confused as to why this wouldn't be plain old annual time. And then model_year should also be a trivial "constant" type.

Reply:
Per the meeting today, `TrivialTime` will become `NoOpTime`